cfme/v2v/migrations.py

```diff
@@ -211,17 +210,21 @@ def fill(self, values):
                         ...
                     }
         """
-        for cluster in values:
+        for cluster in (values if isinstance(values, dict) else []):
             if self.cluster_selector.is_displayed:
                 self.cluster_selector.fill(cluster)
             for mapping in values[cluster]['mappings']:
                 self.source_networks.fill(mapping['sources'])
                 self.target_networks.fill(mapping['target'])
                 self.add_mapping.click()
         was_change = not self.mappings_tree.is_empty
-        if was_change:
-            self.logger.info("Fill operation was successful, click Next.")
+        # Cancel button is the common button on all pages so
```

Comment:
TODO:

I believe these actions would be more appropriate in after_fill, since its not part of actually filling these form values, and before_fill/after_fill would allow for each modal page to wait for elements before starting to fill, and to click next/save where appropriate.

Reply:
@mshriver :allow me to do this in next PR please where I am adding one more test case . 
Will make this change too .